examples/tcp-socket-listener-client/tcp_socket_client.out

```diff
@@ -2,8 +2,8 @@
 # `.bal` file and use the `ballerina run` command.
 $ ballerina run tcp_socket_client.bal
 
-# `onConnect` invokes and prints the remote server port once the connection to the server is successful.
+// `onConnect` invokes and prints the remote server port once the connection to the server is successful.
```

Comment:
Will `//` render the comments separately with the .out files? We seem to have used `#` in other .out files too, should we update all if `//` works?

Reply:
I think `#` has been used to indicate that a particular line is not a command, but a comment. So I think it is better to use the `#` in all the places.

Reply:
If we use // instead of # in when rendering in site, they will appear as comment boxes towards the right hand side of the page. (Similar to the code comments) But yes, we need to change all the places if we agreed to do that change. Hence will revert this change and we can fix all later if needed